pkg/minikube/cluster/cluster.go

```diff
@@ -618,3 +618,17 @@ func GetServiceURLs(api libmachine.API, namespace string, t *template.Template)
 
 	return serviceURLs, nil
 }
+
+//Create the host directories in the virtual machine if not present
+func EnsureHostDirectories(api libmachine.API, dirs []string) (error) {
```

Comment:
+1 agree. This is actually minishift 'specific' knowledge, and not so much cluster. It is something that needs to happen...